### ISSUE TITLE
[Formatting Bug]: Chaining method issue

### ISSUE THREAD

**Issue by dicktornfeldt** [shufo/vscode-blade-formatter#877]
### Platform

MacOS

### Template before formatting

```blade
<flux:tab :data-selected="request()->routeIs('meals*')" href="{{ route('meals.index') }}">
```

### Template after formatting

```blade
<flux:tab :data-selected="request() - > routeIs('meals*')" href="{{ route('meals.index') }}">
```

### Expected Behaviour

The arrow between request() and routeIs() should not get expanded

**Comment by dicktornfeldt** [comment]
I understand that it's because im using Flux components instead of basic Laravel components after reading: #506

**Comment by ferremek** [comment]
@dicktornfeldt I have the same issue with Flux components, did you find a fix for this? Because I would like to use VS Code (actually Cursor) instead of PhpStorm but I have issues with the formatting of Blade files.

Because for example I would like to use the following:

```
<flux:button size="sm" icon="chevron-up" :disabled="$item->user_id === auth()->id()">
    {{ $upvoteCount }}
</flux:button>
```

But after saving it formats like this
```
<flux:button size="sm" icon="chevron-up" :disabled="$item - > user_id === auth() - > id()">
    {{ $upvoteCount }}
</flux:button>
```

Thanks!

**Comment by jpmurray** [comment]
Yeah, I had to disable the extension alltogether because it made it unusable to have to deal with those spacing :(

**Comment by dadastudio** [comment]
You can disable formating for a line of code just like this:
```
{{-- blade-formatter-disable --}}
<flux:accordion.item :expanded="$loop->first">
{{-- blade-formatter-enable --}}
```

**Comment by hallinmattias** [comment]
For strings the problem can be bypassed by passing the variable in curly brackets, `icon="{{ $item->icon }}"` rather than `:icon="$item->icon"`. Does not seem to work for booleans like `disabled="{{ $item->disabled }}"` though.

**Comment by Ceneka** [comment]
Same problem here, I solved using @ instead of :
it works but its not "the correct way"

<flux:navlist.item ... @current="request()->routeIs('admin.dashboard')" ...>

seems to work because the formatter it's ignoring it

**Comment by rizkidarmawan21** [comment]
I got same error,

```
<flux:navlist.item :href="tenant_route('tenant.settings.business_profile')" :current="request() -> routeIs('tenant.settings.business_profile')" wire:navigate>
       { __('Business Profile') }}
</flux:navlist.item>
```

And after formated
```
<flux:navlist.item :href="tenant_route('tenant.settings.business_profile')" :current="request() - > routeIs('tenant.settings.business_profile')" wire:navigate>
       { __('Business Profile') }}
</flux:n

**Comment by ahmadfahmidj** [comment]
i got this error to in livewire starter kit

**Comment by shufo** [comment]
Sorry for late reply.
v0.25.0 released just now introduces `Component Prefix` option that allow specify custom component name like `flux`

<img width="640" height="223" alt="Image" src="https://github.com/user-attachments/assets/1d128ea9-8992-4fb3-b847-67ed5c33230f" />

Please upgrade extension to latest version and specify this option as `x-,livewire,flux`

This option will preventing unexpected formatting of component attribute.
